setup.cfg

```
@@ -71,6 +71,7 @@ install_requires =
   # ansible-lint does not list ansible as direct dependency in order to
   # allow user to choose his target version. Be sure that you mention one
   # of the extras or install a specific version of ansible yourself.
+  ansible-compat >=0.3.0, <0.4.0
```

Let's try this, it's equivalent to the dep spec above but that is semantically closer to SemVer expectations:
```suggestion
  ansible-compat ~= 0.3.0
```

---

In fact I will completely remove the capping and we will add it only when we will break the API with compat. We know very well who are to be the consumers. That is because capping it too soon it would require extra maintenance for us.